### PR TITLE
chore: Add transition for UIKit Modal

### PR DESCRIPTION
**Pull request by Chef-Yogi**
https://user-images.githubusercontent.com/99634186/157085989-a2b6a76e-27bc-43dd-a733-14a44d5709fa.mp4

https://user-images.githubusercontent.com/99634186/157089224-e66c69d2-7d16-4133-bea1-c76a851555dc.mp4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
   "devDependencies": {
     "@next/eslint-plugin-next": "^12.0.10",
     "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
+    "@testing-library/react": "^11.2.5",
```
Comment (chef-jojo) [inline package.json:98]: Instead of downgrade this, maybe try upgrade uikit one

Comment (Chef-Yogi) [reply package.json]: Done

**Comment by chef-jojo** [comment]
Can it be a little bit be faster animation?

**Comment by Chef-Yogi** [comment]
Sure

**Comment by Chef-Yogi** [comment]
@chef-jojo Done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**